pkg/openstack/ovn.go

```diff
@@ -365,9 +366,10 @@ func ReconcileOVNController(ctx context.Context, instance *corev1beta1.OpenStack
 		certRequest := certmanager.CertificateRequest{
 			IssuerName: instance.GetOvnIssuer(),
 			CertName:   fmt.Sprintf("%s-ovndbs", "ovncontroller"),
+			// Cert needs to be valid for the individual pod services in the daemonset so make this a wildcard cert
 			Hostnames: []string{
-				fmt.Sprintf("%s.%s.svc", serviceName, instance.Namespace),
-				fmt.Sprintf("%s.%s.svc.%s", serviceName, instance.Namespace, dnsSuffix),
+				fmt.Sprintf("%s*.%s.svc", serviceName, instance.Namespace),
+				fmt.Sprintf("%s*.%s.svc.%s", serviceName, instance.Namespace, dnsSuffix),
```

Comment:
will this cause an immediate restartof the OVN Controller when openstack-operator is updated? If so should we add code such that it only occurs during the minor update window?